scripts/ci/docker/build_docker_visit_ci.py

```diff
@@ -0,0 +1,100 @@
+# Copyright (c) Lawrence Livermore National Security, LLC and other VisIt
```

Comment:
Just curious...I know you don't like depth of `src/tools/dev/scripts` but why are we starting a new top-level `scripts` dir?

Reply:
This scripts did has existed for a little bit (back with circle ci support).
It was always my intent to have a top level scripts dir, I communicated it wrong some how when you helped move things. 

cd'ing into the deep dir tree isn't to fun :-(

Reply:
I think this makes for confusion between `scripts` we have for `dev` purposes elsewhere. You recommend moving entirety out from `src/tools/dev`? I guess I don't fully appreciate the issue(s) here as `cd /foo` and `cd /foo/bar/gorfo` differ, AFAICT, only in number of characters

Reply:
I don't think we need to change with this pr.

I think my hangup/confusion is that every other repo I develop for has a top level `scripts` dir and it's convenient. 

`src/tools/dev` feels hidden to me, maybe we can discuss at a meeting to see folks thoughts?

Reply:
> maybe we can discuss at a meeting to see folks thoughts?

Sure. Will consider this resolved here though.